freqtrade/optimize/backtesting.py

```diff
@@ -350,8 +350,35 @@ def _get_close_rate(self, sell_row: Tuple, trade: LocalTrade, sell: SellCheckTup
         else:
             return sell_row[OPEN_IDX]
 
+    def _get_adjust_trade_entry_for_candle(self, trade: LocalTrade, row: Tuple
+                                           ) -> LocalTrade:
+
+        # If there is any open orders, wait for them to finish.
+        for order in trade.orders:
+            if order.ft_is_open:
+                return trade
```

Comment:
Backtesting does not have logic to keep an order open (orders are assumed to always fill immediately).

As such, i think it's save to omit this whole block

Reply:
Fixed in next commit.